plugin-modernizer-cli/src/main/java/io/jenkins/tools/pluginmodernizer/cli/Main.java

```diff
@@ -31,6 +31,9 @@ public static void main(final String[] args) {
     @Option(names = {"-r", "--recipes"}, required = true, description = "List of Recipes to be applied.", split = ",", parameterConsumer = CommaSeparatedParameterConsumer.class)
     private List<String> recipes;
 
+    @Option(names = {"-dr", "--dry-run"}, description = "Perform a dry run without making any changes.")
```

Comment:
Short option of 2 letter looks strange to me. I think some tools like Git use the -n option for dry run